api/pom.xml

```diff
@@ -31,6 +31,14 @@
   </build>
 
   <dependencies>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-engine</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.junit.platform</groupId>
+      <artifactId>junit-platform-runner</artifactId>
```

Comment:
Make both of these have `<scope>test</scope>`. This way the jars won't be included in the actual application, only for the test.